src/globus_cli/parsing/commands.py

```diff
@@ -48,6 +50,19 @@ def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
 
 
+class GlobusCommandEnvChecks(GlobusCommand):
+    def invoke(self, ctx):
+        try:
+            env_interactive()
+        except ValueError as e:
+            click.echo(
+                f"couldn't parse GLOBUS_CLI_INTERACTIVE environment variable: {e}",
```

Comment:
I can picture wanting to apply this to other things, in which case the error message might need to be more generic, or `env_interactive` might want to raise its own custom error, but this seemed an acceptable first pass.

Reply:
Yeah, we can refine it over time if necessary.

e.g. I almost suggested that we put the env var definition in a dedicated module. But that's not worth it for only one value.